Makefile

```diff
@@ -17,7 +17,7 @@ BINARY_NAME=network-operator
 PACKAGE=network-operator
 ORG_PATH=github.com/Mellanox
 REPO_PATH=$(ORG_PATH)/$(PACKAGE)
-GOPATH=$(CURDIR)/.gopath
+GOPATH?=$(CURDIR)/.gopath
```

Comment:
make clean will delete it, is that desirable in case of externally provided GOPATH ?

Reply:
Good point. Honestly, I don't care about it at this point.